.github/ISSUE_TEMPLATE/request--new-membership-for--your-gh-handle-.md

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
I notice that k8s does this stuff on the kubernetes/org repo, instead of on the main source repo.

Should we also setup an org repo for this purpose? That would provide instant triaging of issues. Seems like a good long-term strategy

@rshriram @louiscryan @dcberg

Reply:
Yes I noticed that too.  Given we already have a community repo, we could use that... or do it in a new org repo.